test/runtests.jl

```diff
@@ -19,18 +20,21 @@ using OPFGenerator
 const IPOPT_SOLVER = JuMP.optimizer_with_attributes(Ipopt.Optimizer, "linear_solver" => "mumps", "print_level" => 1, "tol" => 1e-6)
 const CLRBL_SOLVER = JuMP.optimizer_with_attributes(Clarabel.Optimizer, "verbose" => true)
 const CLRBL128_SOLVER = JuMP.optimizer_with_attributes(Clarabel.Optimizer{Float128}, "verbose" => true)
+const SCS_SOLVER = JuMP.optimizer_with_attributes(SCS.Optimizer, "eps_abs" => 1e-6, "eps_rel" => 1e-6, "verbose" => 1)
 const HIGHS_SOLVER = JuMP.optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => true)
 
 const OPT_SOLVERS = Dict(
     OPFGenerator.ACOPF             => IPOPT_SOLVER,
     OPFGenerator.SOCOPFQuad        => IPOPT_SOLVER,
     OPFGenerator.SOCOPF            => CLRBL_SOLVER,
+    OPFGenerator.SDPOPF            => SCS_SOLVER,
     OPFGenerator.DCOPF             => HIGHS_SOLVER,
     OPFGenerator.EconomicDispatch  => HIGHS_SOLVER,
 )
 
 
 const PGLIB_CASES = ["14_ieee", "30_ieee", "57_ieee", "89_pegase", "118_ieee"]
+const PGLIB_CASES_SDP = ["14_ieee", "30_ieee"]
```

Comment:
I'd suggest even removing the 30-bus system here.
We can add a separate test for parallel branches